homeassistant/components/crownstone/strings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "title": "Crownstone",
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Comment:
Why is only a single instance allowed? I didn't see any code that required this.

Reply:
This has to do with the Crownstone architecture. A config entry represents a "Sphere". A Sphere could be seen a house, or office. Since a single HA instance represents a house as well, it would not be logical to allow for multiple Spheres per HA instance.

Reply:
I think it's totally reasonable to be able to control my office space before I arrive at work or vice versa. We should not make that decision as developers. We should only limit to a single instance if a second instance is not necessary, like with Cast, which scans the network and find all.

Reply:
There are some difficulties when trying to allow for multiple spheres in a HA instance.

Crownstone communicates through a BLE mesh network and uses the smartphone of the user as gateway to provide the data to a cloud server;  This means, if you want to switch a Crownstone that is in a different sphere, your smartphone would have to be physically there to be able to provide the data from to cloud to the BLE mesh, or in the other direction, send sensor data from the BLE mesh to the cloud server. A hub device is currently in development to help resolve that issue, but it is not ready yet.

Reply:
Sure, so those entities can become unavailable, just like what happens if the phone leaves the "sphere" now. That is unrelated to the number of spheres that can be configured.